schema/google/showcase/v1beta1/echo.proto

```diff
@@ -163,6 +173,20 @@ message PagedExpandRequest {
   string page_token = 3;
 }
 
+// The request for the PagedExpandLegacy method. New APIs should NOT
+// use this pattern; use PagedExpandRequest instead.
+message PagedExpandLegacyRequest {
+  // The string to expand.
+  string content = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // The amount of words to returned in each page.  This is a legacy, non-standard pattern that
+  // violates http://https://google.aip.dev/158
```

Comment:
Nit: can you wrap the reasoning/commentary in an aip.dev/not-precedent comment please?

Reply:
Good idea! I forgot about that.